man/systemd.link.xml

```diff
@@ -80,6 +80,14 @@
           <programlisting>MACAddress=01:23:45:67:89:ab 00-11-22-33-44-55 AABB.CCDD.EEFF</programlisting></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>PermanentMACAddress=</varname></term>
+        <listitem>
+          <para>A whitespace-separated list of hardware's permanent addresses. Use full colon-, hyphen- or dot-delimited hexadecimal.
+          This option may appear more than once, in which case the lists are merged. If the empty string is assigned to this option, the list
+          of hardware addresses defined prior to this is reset.</para>
```

Comment:
there should be a comment here what the difference between MACAddress= and PermanentMACAddress= is...